modules/ROOT/pages/deployment-options.adoc

```diff
@@ -0,0 +1,152 @@
+|===
+
+== Deployments and Components
+Anypoint Platform control plane and runtime components are listed in the following tables with their 
+availability across deployment options. Deployment options include:
+
+* US cloud
+The complete Anypoint Platform. All data is hosted within the US.
+* EU cloud
+Includes all US cloud components, but differs from the US cloud in that all data is hosted within the EU. Refer to xref:eu-control-plane::index.adoc[About the EU Control Plane] for additional information.
+* MuleSoft Government Cloud
+FedRAMP moderate authorized version of US cloud. Refer to xref:gov-cloud::index.adoc[About Government Cloud] for additional information.
+* Anypoint Platform Private Cloud Edition (Anypoint Platform PCE)
+A single-tenant version of US cloud managed by your IT team. Anypoint Platform PCE supports on-premises and cloud installation and is specifically designed for customers with strict regulatory or compliance requirements that limit the use of cloud solutions. Refer to xref:private-cloud::index.adoc[About Anypoint Platform Private Cloud Edition] for additional information.
+
+Refer to https://www.mulesoft.com/anypoint-pricing for Anypoint Platform component availability based on your subscription.
+
+=== Control Plane Components and Availability Across Deployment Options
+[%header,cols="5*a"]
+|===
+| Anypoint Platform Component | US Cloud | EU Cloud | MuleSoft Government Cloud | Anypoint Platform PCE
+| API Designer | Y | Y | Y | Y
+| Studio | Y | Y | Y | Y
+| Flow Designer | Y | Y | N | N
+| Exchange | Y | Y | Y | Y
+| API Community Manager* | Y | Y | N | N
+| Access management | Y | Y | Y | Y
+| Analytics | Y | Y | N | N
+| API Manager | Y | Y | Y | Y
+| Runtime Manager | Y | Y | Y | Y (excluding Insights)
+| Monitoring | Y | Y | N | N
+| Secrets manager | Y | Y | N | N
+| Visualizer | Y | Y | N | N
+|===
+
+*Community Manager is powered by the Salesforce platform.
+
+=== Runtime Plane Options and Deployments
+For the Anypoint Platform runtime plane, you have three options: CloudHub, Runtime Fabric, 
+or on-premises Mule runtime engines. Hybrid environments include both CloudHub and on-premises Mule runtime engine functionality.
+
+* CloudHub
++
+With CloudHub, MuleSoft hosts and manages the infrastructure. Enterprise-grade capabilities, such as 99.99% uptime, one-click scalability, and automatic updates, are provided as a service.
+* Runtime Fabric
++
+Use Anypoint Runtime Fabric if you have requirements to deploy Mule runtimes on your own infrastructure, whether it’s in a private IaaS (Microsoft Azure or AWS) or in an on-premises data center. You can still reap the benefits of CloudHub, such as horizontal scalability and zero downtime redeployments, while keeping your runtimes on-premises.
+* On-premises Mule runtime engines
++
+Use standalone Mule runtime engines when you want to host and manage your own infrastructure.
+
+==== Runtime Plane Availability Across Deployment Options
```

Comment:
We want to avoid nesting information down to a fourth level.
In cases such as these, see if you cannot reorganize to avoid this. Sometimes that involves dividing or promoting topics.

Reply:
So...this must be one of "the following tables" referred to above. There has been a lot between the intro and this first table.